lms/static/js/instructor_dashboard/membership.js

```diff
@@ -463,11 +463,26 @@ such that the value can be defined later than this assignment (file load order).
             if (successes.length && dataFromServer.action === 'add') {
                 // Translators: A list of users appears after this sentence;
                 renderList(gettext('These users were successfully added as beta testers:'), (function() {
-                    var j, len1, results;
+                    var j, k, len1, results, inActive, message, inActiveUsers;
                     results = [];
+                    inActiveUsers = [];
+                    inActive = false;
                     for (j = 0, len1 = successes.length; j < len1; j++) {
                         sr = successes[j];
-                        results.push(sr.identifier);
+                        if (sr.is_active) {
+                            results.push(sr.identifier);
+                        } else {
+                            inActiveUsers.push(sr.identifier);
+                            inActive = true;
+                        }
+                    }
+                    if (inActive) {
+                        message = 'These users could not be added as beta ' +
+                            'testers because their accounts are not yet activated:';
+                        results.push(message);
+                    }
+                    for (k = 0; k < inActiveUsers.length; k++) {
+                        results.push(inActiveUsers[k]);
                     }
                     return results;
```

Comment:
ah, I see why you were adding the active users to `results`. Disregard my last comment then :)